### PR TITLE
[IBCSPRT-18] Remove the need for synapse authentication

### PR DESCRIPTION
**Pull request by thomasyu888**
This PR won't currently work because the publicly available annotation table has not been migrated into staging so currently on the staging site - it is viewed as a resource that can only be downloaded when logged in.

An example of a table that can be accessed without login:

```
import synapseclient
syn = synapseclient.Synapse()
syn.setEndpoints(**synapseclient.client.STAGING_ENDPOINTS)
syn.tableQuery("select * from syn16857542").asDataFrame()
```

### CODE REVIEW

File: server.R [hunk 1]
```diff
@@ -1,46 +1,52 @@
 server <- function(input, output, session) {
   syn <- synapse$Synapse()
-  session$sendCustomMessage(type = "readCookie", message = list())
+  syn$setEndpoints(
```
Comment (thomasyu888) [inline server.R]: Remove this later

File: global.R [hunk 2]
```diff
@@ -6,6 +6,7 @@ suppressPackageStartupMessages(library("reticulate"))
 suppressPackageStartupMessages(library("shiny"))
 
 ## Load synapse client
+reticulate::use_condaenv("synapse")
```
Comment (thomasyu888) [inline global.R]: Remove this later

File: README.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 # AMP-AD Metadata Dictionary
 
+
```
Comment (thomasyu888) [inline README.md]:
```suggestion
```

**Comment by thomasyu888** [comment]
@avanlinden once Synapse gets updated this weekend - you can pull this branch in.  You should be able to launch the application now without needing Synapse login.

**Comment by avanlinden** [comment]
Ugh, yet another repository where everyone with write access has left Sage. I'll ask IT to make me an admin and then merge this PR.

**Pull request merged**